cloudfoundry-operations/src/main/java/org/cloudfoundry/operations/organizationadmin/DefaultOrganizationAdmin.java

```diff
@@ -146,6 +161,21 @@ public DefaultOrganizationAdmin(CloudFoundryClient cloudFoundryClient) {
                 .build());
     }
 
+    private static Mono<UpdateOrganizationQuotaDefinitionResponse> requestUpdateOrganizationQuota(CloudFoundryClient cloudFoundryClient, String quotaDefinitionId, Integer instanceMemoryLimit,
+                                                                                                  Integer memoryLimit, String newName, Boolean nonBasicServicesAllowed, Integer totalRoutes,
+                                                                                                  Integer totalServices) {
+        return cloudFoundryClient.organizationQuotaDefinitions()
```

Comment:
@antechrestos Actually, according to the request trace that contents of the first request (to get the organization quota, not just the quota id) are then used as the "defaults" for the subsequent call.  Notice the instance memory limit field in the following trace:

```
zapp 106155592-update-quota-operations: CF_TRACE=true cf update-quota ben

REQUEST: [2016-04-21T06:45:30-07:00]
GET /v2/quota_definitions?q=name%3Aben HTTP/1.1
Host: api.local.pcfdev.io
Accept: application/json
Authorization: [PRIVATE DATA HIDDEN]
Content-Type: application/json
User-Agent: go-cli 6.17.0+5d0be0a / darwin



RESPONSE: [2016-04-21T06:45:30-07:00]
HTTP/1.1 200 OK
Content-Length: 756
Content-Type: application/json;charset=utf-8
Date: Thu, 21 Apr 2016 13:45:28 GMT
Server: nginx
X-Content-Type-Options: nosniff
X-Vcap-Request-Id: 868e1e4a-005f-4c7d-5e87-7bde25d9624c
X-Vcap-Request-Id: 0311f10d-8e39-48b3-6cc1-3810508f1841::5c4f4772-79ff-4df2-bc3c-26b553a1f157

{
  "total_results": 1,
  "total_pages": 1,
  "prev_url": null,
  "next_url": null,
  "resources": [
    {
      "metadata": {
        "guid": "8339420b-ca96-480c-9b76-b337bb1e35c7",
        "url": "/v2/quota_definitions/8339420b-ca96-480c-9b76-b337bb1e35c7",
        "created_at": "2016-04-21T13:42:12Z",
        "updated_at": "2016-04-21T13:45:21Z"
      },
      "entity": {
        "name": "ben",
        "non_basic_services_allowed": false,
        "total_services": 0,
        "total_routes": 0,
        "total_private_domains": -1,
        "memory_limit": 0,
        "trial_db_allowed": false,
        "instance_memory_limit": 5120,
        "app_instance_limit": -1,
        "app_task_limit": -1,
        "total_service_keys": -1
      }
    }
  ]
}
Updating quota ben as admin...

REQUEST: [2016-04-21T06:45:30-07:00]
PUT /v2/quota_definitions/8339420b-ca96-480c-9b76-b337bb1e35c7 HTTP/1.1
Host: api.local.pcfdev.io
Accept: application/json
Authorization: [PRIVATE DATA HIDDEN]
Content-Type: application/json
User-Agent: go-cli 6.17.0+5d0be0a / darwin

{"guid":"8339420b-ca96-480c-9b76-b337bb1e35c7","name":"ben","memory_limit":0,"instance_memory_limit":5120,"total_routes":0,"total_services":0,"non_basic_services_allowed":false,"app_instance_limit":-1}

RESPONSE: [2016-04-21T06:45:30-07:00]
HTTP/1.1 201 Created
Content-Length: 565
Content-Type: application/json;charset=utf-8
Date: Thu, 21 Apr 2016 13:45:28 GMT
Server: nginx
X-Content-Type-Options: nosniff
X-Vcap-Request-Id: 7f5f9059-5ecf-4c1a-6fbe-f413498dad95
X-Vcap-Request-Id: 5c1b2c8e-8975-4516-706b-f046ee60a9a4::fa39d66f-7842-4218-9ce9-2b50383874d0

{
  "metadata": {
    "guid": "8339420b-ca96-480c-9b76-b337bb1e35c7",
    "url": "/v2/quota_definitions/8339420b-ca96-480c-9b76-b337bb1e35c7",
    "created_at": "2016-04-21T13:42:12Z",
    "updated_at": "2016-04-21T13:45:28Z"
  },
  "entity": {
    "name": "ben",
    "non_basic_services_allowed": false,
    "total_services": 0,
    "total_routes": 0,
    "total_private_domains": -1,
    "memory_limit": 0,
    "trial_db_allowed": false,
    "instance_memory_limit": 5120,
    "app_instance_limit": -1,
    "app_task_limit": -1,
    "total_service_keys": -1
  }
}
OK
```